sim/utils.py

```diff
@@ -11,17 +11,22 @@ def run(cmd, *args, **kwargs):
             cwd (str): Set surr
 
         Returns:
-            None
+            None: if capture_output == False (default)
+            process' stdout (str): if capture_output == True 
     """
     # TODO: store stdout to a file
-    returncode = subprocess.run(cmd, *args, **kwargs).returncode
+    ps = subprocess.run(cmd, *args, capture_output=capture_output, **kwargs)
 
+    returncode = ps.returncode
     if returncode:
         raise Exception("""*****
             %s
             cmd failed with exit code %i
           *****""" % (cmd, returncode))
 
+    if capture_output:
```

Comment:
so this could be
`if "capture_output" in kwargs and kwargs["capture_output"]`

Reply:
Ah yes ok, nice!